src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -339,6 +340,13 @@ namespace HeatBalanceManager {
 		// Get the heat balance input at the beginning of the simulation only
 		if ( ManageHeatBalanceGetInputFlag ) {
 			GetHeatBalanceInput(); // Obtains heat balance related parameters from input file
+
+			// Surface octree setup
+			//  The surface octree holds live references to surfaces so it must be updated
+			//   if in the future surfaces are altered after this point
+			surfaceOctree.init( DataSurfaces::Surface ); // Set up surface octree
```

Comment:
So does this look right:
```cpp
if ( TotSurfaces >= DaylightingManager::octreeCrossover ) { // Octree can be active  
   if ( GetNumObjectsFound( "Daylighting:Controls" ) > 0 ) { // Daylighting is active
      surfaceOctree.init( DataSurfaces::Surface ); // Set up surface octree          
   }                                                                                 
}                                                                                    
```

I dropped the DELight test since I think we are saying that DELight doesn't necessarily trigger use of DaylightingManager.